src/main/java/io/phdata/streamliner/schemadefiner/mapper/SnowflakeMapper.java

```diff
@@ -96,25 +123,37 @@ private static SnowflakeTable mapSchemaCrawlerTable(Table table, Snowflake desti
 
     return new SnowflakeTable(
         table.getName(),
-        getSnowflakeTableName(table.getName(), destination),
+        getSnowflakeTableName(table.getName()),
         table.getRemarks(),
         primaryKey,
         null,
         null,
         null,
-        new FileFormat(table.getName(), "PARQUET"),
+        createFileFormat(table.getName()),
         mapSchemaCrawlerColumnDefinition(columns));
   }
 
-  public static String getSnowflakeTableName(String tableName, Snowflake destination) {
+  private FileFormat createFileFormat(String tableName) {
+    if (attributeRetriever == null) {
+      return new FileFormat(tableName, "PARQUET");
+    } else if (attributeRetriever instanceof HiveAttributeRetriever) {
```

Comment:
There is only a HiveAttributeRetriever right? So this condition is always true

Reply:
I think the `AttributeRetriever` interface isn't necessary, there's only one and will only be one implementation

Reply:
For the moment we have only one. But I have added the `AttributeRetriever` interface thinking that this will be helpful if we have to add similar stuff for other db.

Reply:
I'd probably leave this out myself (YAGNI https://martinfowler.com/bliki/Yagni.html)